private/simplesaml/config/authsources.php

```diff
@@ -1,8 +1,8 @@
 <?php
 
-$idp = (isset($_SERVER['PANTHEON_ENVIRONMENT']) && $_SERVER['PANTHEON_ENVIRONMENT'] == 'live')
-  ? 'https://shibidp.cit.cornell.edu/idp/shibboleth'
-  : 'https://shibidp-test.cit.cornell.edu/idp/shibboleth';
+$idp = (preg_match('(test|dev|localhost|lando|lndo)', $_SERVER['SERVER_NAME']) === 1)
+    ? 'https://shibidp-test.cit.cornell.edu/idp/shibboleth'
+    : 'https://shibidp.cit.cornell.edu/idp/shibboleth';
```

Comment:
We already have access to `$_ENV` thanks to simplesaml/config/config.php, so I think we can just use that and keep the "only if" quality the original?
```suggestion
$idp = ($_ENV['APP_ENV'] == 'live')
    ? 'https://shibidp.cit.cornell.edu/idp/shibboleth'
    : 'https://shibidp-test.cit.cornell.edu/idp/shibboleth';
```